sw/host/opentitanlib/src/transport/cw310/mod.rs

```diff
@@ -182,6 +182,9 @@ impl Transport for CW310 {
 
     fn dispatch(&self, action: &dyn Any) -> Result<Option<Box<dyn Serialize>>> {
         if let Some(fpga_program) = action.downcast_ref::<FpgaProgram>() {
+            // Open the console UART.  We do this first so we get the receiver
+            // started and the uart buffering data for us.
+            let uart = self.uart("0")?;
```

Comment:
Ideally, the test log would begin while the chip is held in the initial reset, so there is no extraneous info from a prior run (not anything lost from this one). I'm not sure if that's actually possible, though.